### ISSUE TITLE
MaterialDesignBodyLight appearing too light?

### ISSUE THREAD

**Issue by EdonGashi** [ButchersBoy/MaterialDesignInXamlToolkit#408]
While trying to implement simple [alerts](https://material-design.storage.googleapis.com/publish/material_v_8/material_ext_publish/0Bzhp5Z4wHba3cGUwa0F5ekUwVms/components_dialogs_usage1.png) I noticed that my text was appearing lighter than expected.

Looking at the light theme dictionary there is "MaterialDesignBodyLight" Color="#89000000"

89h / FFh = 137 / 255 = 0.5372 which is very close to 0.54

Theoretically black * 0.54 opacity should equal MaterialDesignBodyLight; yet I get noticeable difference with this simple experiment:

```xaml
<TextBlock Foreground="{DynamicResource MaterialDesignBodyLight}"
           FontSize="20"
           Text="Test 123456" />
<TextBlock Foreground="Black"
           Opacity="0.54"
           FontSize="20"
           Text="Test 123456" />
```

![light](https://cloud.githubusercontent.com/assets/12145268/15952322/f3b2ecc8-2ebf-11e6-8047-250f6bcee741.png)

The left stripe is a black photoshop layer on 54% opacity (#737373). Setting MaterialDesignBodyLight to #B0000000 gives me a closer match on those two:

![light2](https://cloud.githubusercontent.com/assets/12145268/15952376/93e52062-2ec0-11e6-9e81-9e0288f69cab.png)

Can anyone else reproduce this issue? Is this something internal to how wpf handles color alpha vs control opacity?

**Comment by ButchersBoy** [comment]
Hi, can you try ascertaining the absolute RGB value without alpha, and using that in the MaterialDesignBodyLight definition.

If it looks OK, and we do the same for the dark theme, I can consider a PR for this.

**Comment by EdonGashi** [comment]
Overlaying a #000000 at 54% opacity layer over #FAFAFA (MaterialDesignPaper) in photoshop gives me #737373. I get the same value by overlaying two rectangles in WPF; the first at #FFFAFAFA, the other at #FF000000 with Opacity=54%. 

Overlaying two wpf rectangles (both with control opacity 100%) with fill #89000000 over #FFFAFAFA gives me #747474, which is correct considering the opacity rounds to 53.7%.

It seems that only TextBlocks blend incorrectly. They give a #A0A0A0 color. I also tested this scenario in a clean project without any references. 

While typing this I discovered something: when font size is below a threshold the text displays at #A0A0A0, setting a large value it goes to the expected #747474...

Both textblocks below have Foreground="{DynamicResource MaterialDesignBodyLight}":

![fontsize](https://cloud.githubusercontent.com/assets/12145268/15968148/0c4d5848-2f2b-11e6-9ce6-ed9caa275990.png)

Setting a fixed RGB defies color specifications anyway, so I guess it's best to use Black text with overriden control opacity.

```
Use opacity instead of grey

Black or white text that is transparent remains legible and vibrant against background color changes. This makes it more flexible than grey text in the same contexts.
```
